.github/workflows/borg_integration_tests.yml

```
@@ -0,0 +1,94 @@
+            client-version: 1.4.1
+            server-version: 1.4.1
+            os-type: macos
+
+    name: ${{ matrix.os-type }} - ${{ matrix.os }} - Client ${{ matrix.client-version }} / Server ${{ matrix.server-version }}
+    
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup Go
+        uses: actions/setup-go@v4
```

actionlint (1.7.7)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)